### DISCUSSION TITLE
Mastodon PM-System connection to zammad or OTRS

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#17087] [General]
Have there been any thoughts on how Mastodon PM system could be connected to systems like OTRS or Zammad (free and open source ticket, support and/or helpdesk systems)? 

This could be very exciting especially for larger companies using Mastodon accounts and could be a crucial point for the operation of a Mastodon account or Fediverse account in general - the connection to systems that are already used in the company / association / organization.

@Gargron

**Comment by Gargron** [comment]
What do you mean by PN system?

**Reply by linux-lukas** [reply]
I seem to have misstyped that - instead of PN I meant PM (Private Message). Sorry. Thank you for pointing it out.

**Comment by linux-lukas** [comment]
I could basically also point out a concrete use case: 


A publishing house like the German-language "Spiegel" would like to use Fediverse. For example, it uses the FLOSS ticket system Zammad for contact possibilities via e-mail. 

If Spiegel wants to use Fediverse, it would be advantageous to be able to enter these channels into the ticket system so that the people who are active there do not have to learn ANOTHER system.

I hope it was clear what I meant by the idea.😅

cc: @Gargron
